releases/handbook.md

```diff
@@ -1,193 +1,71 @@
-* Help to configure, monitoring and operate the Working Group's test infrastructure i.e. GitHub Actions and/or AWS Test Infrastructure
-* Help to complete test cases per the test plan
-* Provide test plan results to the Release Team 
-
-### Distribution Representative
-
-**Prerequisites:** Work at the organization that owns the distribution
-
-**Time Commitments:** This role can be completely asynchronous. Most of their work is expected to be in the Distribution testing phase. Ideally they should be involved when the first RC is cut in order to alert early for potential issues.
-
-**Responsibilities:**
-- Expose issues that the distribution owner is facing with the new release
-- Sync with the release team during distro testing phase, we want a nice tight feedback loop there
-
-## Proposed WG Processes
-
-### Releasing
-
-We are moving to a release cadence of **4 months**, in response to our 3 month cycle being tight for the typical set of features we have wanted to release.The release is planned out with 10 weeks of development, with 6 weeks of feature freeze, documentation work, and other processes. At some point there will be a feature freeze, as described below, in the manifests repo. This means that other WGs cannot request the Manifest WG leads to update the repo to change a component’s YAML files, if the update sets a new image with new features.
-
-The above means that WGs, which will be maintaining a list of repos, will only be providing git revisions that only include bug fixes and not features, to the Manifests WG leads.
-
-We recommend adopting a release branch to cope with this form of releasing; we suggest that all WGs follow the process below for releasing new versions of their software:
-
-1. Create a new branch for your new version
-2. Use tags for the release that will be pointing to commits in the created branch
-
-This way a WG can keep on pushing new features to their **master** branch. Once there’s a feature freeze, in manifests repo, they can cherry-pick bug fixes to their new branch and request from the Manifests WG leads to sync the manifests from commits in that branch. This ensures that the newer commits will only include bug fixes.
-
-### Documentation
-
-Working groups should briefly describe what changes to existing [kubeflow.org](https://www.kubeflow.org/) documentation will be required to ensure the docs reflect new features or other software updates. A few bullets identifying which pages in the docs need to change will suffice. These bullets should be added to the Working Group’s GitHub issues describing planned engineering work.
-
-Working Groups should be tracking features for the release, as well as the documentation status of each feature as it’s being developed so that the documentation team can keep track of the documentation work that needs to be done.
-
-## Versioning Policy
```

Comment:
why is it removed the version policy? is there a change on this?
@juliusvonkohout

Reply:
If it is really missing and important enough we can re add it, but the goal is to keep the document as short as possible. In the end we should cover this with a single sentence if needed.